test/integration/foreman_puppet/smartclass_parameter_js_test.rb

```diff
@@ -74,6 +74,7 @@ class SmartclassParameterJSTest < IntegrationTestWithJavascript
       within(:xpath, '//table') do
         click_link 'ssl'
       end
+      page.driver.browser&.file_detector = nil
```

Comment:
Can you add a comment  to the code as well, with comment link to the Redmine? So it's immediately clear what's the reasoning behind this workaround.

Reply:
done